### ISSUE TITLE
Add shuffling in dataloader

### ISSUE THREAD

**Issue by hermancollin** [axondeepseg/sam_myelin_seg_TEM#1]
https://github.com/axondeepseg/sam_myelin_seg_TEM/blob/415b98a0142b5dff71205749b641bc1e96638c84/scripts/bids_utils.py#L70-L87

Order in which samples are loaded should be shuffled.

**Comment by hermancollin** [comment]
Now supported with the custom AxonDataset class, which can be fed to torch.utils.data.DataLoader (see 5dfe03e)
